packages/calling-stateful-client/src/CallCommonDeclarative.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(teams-call) */
+import { CallCommon as CallCommonBeta } from '@azure/communication-calling';
+import { Call } from '@azure/communication-calling';
+import { CallContext } from './CallContext';
+
+/**
+ * @private
+ */
+export type CallCommon = Call | /* @conditional-compile-remove(teams-call) */ CallCommonBeta;
```

Comment:
Please add a comment describing what this black magic is, and why this is needed.

Reply:
This is da real black magic.